software/contrib/quantizer.py

```diff
@@ -107,8 +109,8 @@ def draw(self):
         oled.text('+', self.playing_marks[k][0], self.playing_marks[k][1], self.playing_marks[k][2])
 
         # clear the bottom of the screen and mark the togglable key with a line
-        oled.fill_rect(0, 30, 128, 32, 0)
-        oled.fill_rect(self.enable_marks[self.highlight_note][0], 31, 7, 1, 1)
+        oled.fill_rect(0, OLED_HEIGHT-2, OLED_WIDTH, 2, 0)
```

Comment:
[Question] The third parameter here is changing from 32 to 2, is this correct?

Reply:
Correct; that looks like it was actually a mistake originally; the third parameter is the height of the rectangle, but originally it was starting at row 30 with a height of 32. But the screen is only 32 rows high, so the extra height was off-screen. This correctly changes the rectangle to a height of 2, which is all that's on-screen anyway.

Reply:
Yep, that's what I meant. This explanation makes sense. Thanks.